client/components/async-load/index.jsx

```diff
@@ -6,8 +6,13 @@ import { omit } from 'lodash';
 
 export default class AsyncLoad extends Component {
 	static propTypes = {
+		noPlaceholder: PropTypes.bool,
+		placeholder: PropTypes.node,
 		require: PropTypes.func.isRequired,
-		placeholder: PropTypes.node
+	};
+
+	static defaultProps = {
+		noPlaceholder: false,
```

Comment:
We can just set default prop to be the placeholder component, so if you pass false/null it should work.